english/src/README.md

```diff
@@ -1,18 +1,32 @@
 # @lnp2pBot
 
-On this site you will find information about [@lnp2pBot](https://t.me/lnp2pbot). It is an application to buy and sell bitcoin on Lightning Network with your local currency, without giving your personal data.
+Here you'll find information about [@lnp2pBot](https://t.me/lnp2pbot). This is a Telegram bot for buying and selling bitcoin on the Lightning Network, in a peer-to-peer manner, using your local currency without requiring KYC or providing personal data.
 
 ## Quick Start Guide
 
-1. Download a Lightning Network compatible wallet ([Recomended wallets](./recommended-wallets.md)).
-2. Get a Telegram username.
-3. Start [@lnp2pBot](https://t.me/lnp2pbot)
+1. Download a Lightning Network-compatible wallet ([Recommended Wallets](./recommended-wallets.md)).
+2. Get a Telegram user.
```

Comment:
please change back this to `username` which is exactly what the user needs to do

Reply:
done